references/rpm_ostree_command_ref.adoc

```diff
@@ -0,0 +1,251 @@
+[[rpm-ostree-command-ref]]
+= {product-title} rpm-ostree Command Reference
```

Comment:
This all seems fine to me...I am just uncertain about the big picture here versus the docs that live in rpm-ostree upstream.  

Offhand...it feels like upstream should be more detailed, but this doc covers a high level?  And we link to upstream?

Reply:
@jlebon and I were discussing ideas on how to have these docs co-exist, especially stuff like this that is in the category of 'command reference'.  Mirroring the upstream docs here might be a solution.

I'm happy to keep this high-level and make changes to adjust; the original content was from upstream, so I was just massaging it a bit.  If you think any changes here should be moved to the rpm-ostree repo (or readthedocs), just let me know.

@jberkus Any thoughts on the subject of command reference docs (like this) and where they should live?

Reply:
I opened an issue to talk about this here: https://github.com/projectatomic/atomic-host-docs/issues/95.